_data/nil3sh99/nil3sh.md

```diff
@@ -0,0 +1,5 @@
+name: Nilesh Aggarwal 
+institution: Bharati Vidyapeeth's College of Engineering 
+profile_pic: nilesh.png 

```

Comment:
image file path name seems incorrect, you might want to change it from `nilesh.png ` to `nil3sh.png `